### PR TITLE
Add group details to readme

### PR DESCRIPTION
**Pull request by sky-2002**
This PR adds group details to the readme file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+**Group Name: A-Cube**
```
Comment (SentinelReacon) [inline README.md:3]: @sky-2002  Add repo info like what this repo is for etc, above this line

**Comment by VishnoiAman777** [comment]
Hey Akash this looks good

**Comment by SentinelReacon** [comment]
Starting is good!

File: README.md (hunk 1 above)
Comment (sky-2002) [reply README.md:3]: Sure

**Pull request merged**